### PR TITLE
Prepare release 1.1.1

### PR DESCRIPTION
**Pull request by dmytro-gokun**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.1.1 (2020-05-18)
+* Fix `npm audit` warnings from a dependency.
```
Comment (andrewharvey) [inline CHANGELOG.md:4]:
```suggestion
* Fix `npm audit` warnings from a dependency.
* Fix reversing of coordinates when there are more than 2
```

Comment (andrewharvey) [reply CHANGELOG.md:4]: this is from https://github.com/mapbox/polyline/pull/65

**Comment by andrewharvey** [comment]
@tristen I've tagged a new release https://github.com/mapbox/polyline/releases/tag/v1.1.1 which includes the latest changes. I hope that's okay! This just needs an `npm publish` now.

**Comment by tristen** [comment]
👍 done! https://www.npmjs.com/package/@mapbox/polyline/v/1.1.1

**Pull request merged**